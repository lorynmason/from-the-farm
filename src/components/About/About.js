import React from 'react';

export const About = () => {
  return (
    <main>
      <div className="about">
        <p>
          from the farm is a community based effort to break the traditional food supply chain and connect Farmers and Producers directly to Restaurants and Markets.
        </p>
        <p>
          Please note that from the farm is still in the development phase, which means all data currently on the site is a placeholder, and doesn't represent real producers or products. 
        </p>
        <p>
          If you'd like to contribute code or find out more information about from the farm please visit: <a href="https://github.com/lorynmason/from-the-farm">https://github.com/lorynmason/from-the-farm</a>
        </p>
      </div>
    </main>
  );
};
