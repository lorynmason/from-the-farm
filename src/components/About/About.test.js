import React from 'react';
import { shallow } from 'enzyme';
import { About } from './About';

describe('About', () => {
  it('should match snapshot', () => {
    const wrapper = shallow(<About />);
    expect(wrapper).toMatchSnapshot();
  });
});
