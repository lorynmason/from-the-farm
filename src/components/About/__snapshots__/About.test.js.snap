// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About should match snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <About />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <div
        className="about"
      >
        <p>
          from the farm is a community based effort to break the traditional food supply chain and connect Farmers and Producers directly to Restaurants and Markets.
        </p>
        <p>
          Please note that from the farm is still in the development phase, which means all data currently on the site is a placeholder, and doesn't represent real producers or products.
        </p>
        <p>
          If you'd like to contribute code or find out more information about from the farm please visit: 
          <a>
            https://github.com/lorynmason/from-the-farm
          </a>
        </p>
      </div>,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <p>
            from the farm is a community based effort to break the traditional food supply chain and connect Farmers and Producers directly to Restaurants and Markets.
          </p>,
          <p>
            Please note that from the farm is still in the development phase, which means all data currently on the site is a placeholder, and doesn't represent real producers or products.
          </p>,
          <p>
            If you'd like to contribute code or find out more information about from the farm please visit: 
            <a>
              https://github.com/lorynmason/from-the-farm
            </a>
          </p>,
        ],
        "className": "about",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "from the farm is a community based effort to break the traditional food supply chain and connect Farmers and Producers directly to Restaurants and Markets.",
          },
          "ref": null,
          "rendered": "from the farm is a community based effort to break the traditional food supply chain and connect Farmers and Producers directly to Restaurants and Markets.",
          "type": "p",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Please note that from the farm is still in the development phase, which means all data currently on the site is a placeholder, and doesn't represent real producers or products.",
          },
          "ref": null,
          "rendered": "Please note that from the farm is still in the development phase, which means all data currently on the site is a placeholder, and doesn't represent real producers or products.",
          "type": "p",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              "If you'd like to contribute code or find out more information about from the farm please visit: ",
              <a>
                https://github.com/lorynmason/from-the-farm
              </a>,
            ],
          },
          "ref": null,
          "rendered": Array [
            "If you'd like to contribute code or find out more information about from the farm please visit: ",
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "https://github.com/lorynmason/from-the-farm",
              },
              "ref": null,
              "rendered": "https://github.com/lorynmason/from-the-farm",
              "type": "a",
            },
          ],
          "type": "p",
        },
      ],
      "type": "div",
    },
    "type": "main",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="about"
        >
          <p>
            from the farm is a community based effort to break the traditional food supply chain and connect Farmers and Producers directly to Restaurants and Markets.
          </p>
          <p>
            Please note that from the farm is still in the development phase, which means all data currently on the site is a placeholder, and doesn't represent real producers or products.
          </p>
          <p>
            If you'd like to contribute code or find out more information about from the farm please visit: 
            <a>
              https://github.com/lorynmason/from-the-farm
            </a>
          </p>
        </div>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <p>
              from the farm is a community based effort to break the traditional food supply chain and connect Farmers and Producers directly to Restaurants and Markets.
            </p>,
            <p>
              Please note that from the farm is still in the development phase, which means all data currently on the site is a placeholder, and doesn't represent real producers or products.
            </p>,
            <p>
              If you'd like to contribute code or find out more information about from the farm please visit: 
              <a>
                https://github.com/lorynmason/from-the-farm
              </a>
            </p>,
          ],
          "className": "about",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "from the farm is a community based effort to break the traditional food supply chain and connect Farmers and Producers directly to Restaurants and Markets.",
            },
            "ref": null,
            "rendered": "from the farm is a community based effort to break the traditional food supply chain and connect Farmers and Producers directly to Restaurants and Markets.",
            "type": "p",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Please note that from the farm is still in the development phase, which means all data currently on the site is a placeholder, and doesn't represent real producers or products.",
            },
            "ref": null,
            "rendered": "Please note that from the farm is still in the development phase, which means all data currently on the site is a placeholder, and doesn't represent real producers or products.",
            "type": "p",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                "If you'd like to contribute code or find out more information about from the farm please visit: ",
                <a>
                  https://github.com/lorynmason/from-the-farm
                </a>,
              ],
            },
            "ref": null,
            "rendered": Array [
              "If you'd like to contribute code or find out more information about from the farm please visit: ",
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "https://github.com/lorynmason/from-the-farm",
                },
                "ref": null,
                "rendered": "https://github.com/lorynmason/from-the-farm",
                "type": "a",
              },
            ],
            "type": "p",
          },
        ],
        "type": "div",
      },
      "type": "main",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
